### ISSUE TITLE
Update Cert for HRBlock

### ISSUE THREAD

**Issue by blisteringherb** [DoSomething/infrastructure#75]
The SSL cert for caps.hrblock.com is set to expire on 12/9, and it needs to be updated. The ideal solution would be to use Let's Encrypt with auto-renew so we don't have to worry about it anymore, but there are some issues with that. There are a few entries that are run before all others that are preventing using HAProxy as a standalone webserver to verify the caps URL:  

```
redirect prefix https://longshot-qa.dosomething.org code 301 if { hdr(host) -i longshot-qa.dosomething.org }
redirect prefix https://footlockerscholarathletes.com code 301 if { hdr(host) -i footlockerscholarathletes.com }
redirect prefix https://footlockerscholarathletes.com code 301 if { hdr(host) -i www.footlockerscholarathletes.com }
redirect prefix https://caps.hrblock.com code 301 if { hdr(host) -i caps.hrblock.com }
```

I need to add these entries to allow Let's Encrypt to validate the URL and enable auto renew:  
```
# Test URI to see if its a letsencrypt request
acl letsencrypt-acl path_beg /.well-known/acme-challenge/
use_backend letsencrypt-backend if letsencrypt-acl  

# LE Backend
backend letsencrypt-backend
    server letsencrypt 127.0.0.1:8888
```  
The issue is that because HAProxy always runs `redirect` rules before `use_backend` and sends all requests straight to the Heroku app, Let's Encrypt isn't able to use our standalone server to respond properly to the verification. Are we able to move or remove those entries or accomplish what the purpose of those entries in another way? @sheyd @DFurnes If we can, we'll be able to use the same approach for Footlocker as well.

Here's the command to set up the certs initially:  
```
sudo certbot certonly --standalone -d caps.hrblock.com \
    --non-interactive --agree-tos --email devops@dosomething.org \
    --http-01-port=8888
```

Here's what the auto-renew script will look like for reference:  
```
#!/usr/bin/env bash

# Renew the certificate
certbot renew --force-renewal --tls-sni-01-port=8888

# Concatenate new cert files, with less output (avoiding the use tee and its output to stdout)
bash -c "cat /etc/letsencrypt/live/caps.hrblock.com/fullchain.pem /etc/letsencrypt/live/caps.hrblock.com/privkey.pem > /etc/ssl/demo.scalinglaravel.com/caps.hrblock.com.pem"

# Reload  HAProxy
service haproxy reload  
```

There are some additional setup steps not listed here, but I'm following this tutorial as a guide. https://serversforhackers.com/c/letsencrypt-with-haproxy

**Comment by blisteringherb** [comment]
I went ahead and purchased a cert in the hopes that I could just do the DCV validation and get things rolling. Unfortunately I'm running into issues here as well. I set up HAproxy to respond to the necessary URL under both http and https, and Comodo isn't registering the authorization. Support is reporting that the tool is running into these errors: `HTTP/1.x transport connection broken: malformed HTTP response` and `The server committed a protocol violation.`
